app/src/main/java/com/x8bit/bitwarden/data/autofill/fido2/manager/Fido2CredentialManager.kt

```diff
@@ -11,6 +11,12 @@ import com.x8bit.bitwarden.data.autofill.fido2.model.Fido2ValidateOriginResult
  */
 interface Fido2CredentialManager {
 
+    /**
+     * Returns true when the user has performed an explicit verification action. E.g., biometric
+     * verification, device credential verification, or vault unlock.
+     */
+    var isUserVerified: Boolean
```

Comment:
Dumb question, but for my own knowledge, this is only for the case where the user doesn't have screen lock enabled on their device so they haven't been verified by the time they get to the app, right?

Reply:
Not a dumb question at all. It's not exclusive to device verification, but is only relevant in the context of FIDO operations. It represents a broader verification state that also includes BW methods of authentication like unlocking the vault or entering their MP to view a protected cipher. The expectation is that when we add BW authentication methods to this flow we will update `isUserVerified` based on the result.

Reply:
Awesome thanks!